CHANGELOG.md

```diff
@@ -1,5 +1,220 @@
+* App create wizard support for kustomize apps ([#31](https://github.com/argoproj/argo-cd/issues/31))
+* Merge pull request [#27](https://github.com/argoproj/argo-cd/issues/27) from alexmt/459-app-wizard-improvement
+* Issue [#459](https://github.com/argoproj/argo-cd/issues/459) - Improve application creation wizard
+* Merge pull request [#26](https://github.com/argoproj/argo-cd/issues/26) from alexmt/474-list-apps
+* Merge pull request [#25](https://github.com/argoproj/argo-cd/issues/25) from alexmt/446-loading-error-notification
+* Issue [#446](https://github.com/argoproj/argo-cd/issues/446) - Improve data loading errors notification
+* Merge pull request [#23](https://github.com/argoproj/argo-cd/issues/23) from merenbach/fix-application-card
+* Merge pull request [#22](https://github.com/argoproj/argo-cd/issues/22) from alexmt/443-helm-app
+* Merge pull request [#20](https://github.com/argoproj/argo-cd/issues/20) from alexmt/340-app-events-ui
+* Issue [#406](https://github.com/argoproj/argo-cd/issues/406) - add button to terminate a operation
+* Issue [#402](https://github.com/argoproj/argo-cd/issues/402) - App deployment history don't display parameter overrides
+* Issue [#400](https://github.com/argoproj/argo-cd/issues/400) - Provide a link to swagger UI
+* Issue [#290](https://github.com/argoproj/argo-cd/issues/290) - Cluster list page
+* Issue [#341](https://github.com/argoproj/argo-cd/issues/341) - add refresh button in app view
+* Issue [#337](https://github.com/argoproj/argo-cd/issues/337) - remember my resource filtering preferences
+* Issue [#306](https://github.com/argoproj/argo-cd/issues/306) - UI should allow redeploying most recent successful deployment from history
+* Issue [#352](https://github.com/argoproj/argo-cd/issues/352) -  resource names are almost always truncated
+* Support  option for app sync operation on app details page [#289](https://github.com/argoproj/argo-cd/issues/289)
+* Issue [#231](https://github.com/argoproj/argo-cd/issues/231) - Display pod status on application details page
+* Issue [#286](https://github.com/argoproj/argo-cd/issues/286) - Resource events tab on application details page
+* Issue [#241](https://github.com/argoproj/argo-cd/issues/241) - Repositories list page
+* Issue [#232](https://github.com/argoproj/argo-cd/issues/232) - Resource filtering on Application Details page
+* Issue [#235](https://github.com/argoproj/argo-cd/issues/235) - Allow viewing pod side car container logs
+* Issue [#230](https://github.com/argoproj/argo-cd/issues/230) - Display operation state on application details page
+* Issue [#184](https://github.com/argoproj/argo-cd/issues/184) - Allow downloading of argocd binaries directly from API server
+* Issue [#189](https://github.com/argoproj/argo-cd/issues/189) - switch to Spec.Destination.Server/Namespace fields
+* Issue [#191](https://github.com/argoproj/argo-cd/issues/191) - ArgoCD UI s/rollback/history/ and s/deploy/sync/
```

Comment:
this looks like a bug in generator :)

![image](https://user-images.githubusercontent.com/426437/61811203-d560e280-adf5-11e9-9d9c-7401f6925c8f.png)